### DISCUSSION TITLE
Add indicator that image is only in cloud, not on device

### DISCUSSION THREAD

**Discussion by charliefrance** [ente-io/ente#2829] [Enhancements]
It would be very useful to add an indicator like the "not in cloud" indicator, but for when an image is not on your device.

An indicator that looked like this, but was a device with a line through it instead of a cloud to show that it is not on your device
![Screenshot_20240822_215046_Ente Photos](https://github.com/user-attachments/assets/172c28f2-fb10-4948-9e6c-df2016d7ad97)

Here is an example of what the icon should look like:
![Screenshot_20240822_215219_Photos](https://github.com/user-attachments/assets/a6deb919-d2f4-4a7c-9cff-03e67ac80cf8)

**Comment by NotLe0n** [comment]
This in addition to an option to filter by images not on device would make it much easier to clean my cloud if I deleted the image on my device first.
